1diff --git a/src/modules/CUVID/CMakeLists.txt b/src/modules/CUVID/CMakeLists.txt
index a9fca1e1..03d57dc8 100644
--- a/src/modules/CUVID/CMakeLists.txt
+++ b/src/modules/CUVID/CMakeLists.txt
@@ -48,4 +48,8 @@ if(WIN32)
     target_link_libraries(${PROJECT_NAME} opengl32)
 endif()
 
+if(APPLE)
+    target_link_libraries(${PROJECT_NAME} ${OPENGL_LIBRARY})
+endif()
+
 install(TARGETS ${PROJECT_NAME} LIBRARY DESTINATION ${MODULES_INSTALL_PATH})
commit 5672989e7cb8c08828cdc7ffe18b002498bff9bd
Author: Błażej Szczygieł <mumei6102@gmail.com>
Date:   Mon Aug 7 21:41:38 2023 +0200

    OpenGL: Don't get version on macOS

diff --git b/src/qmplay2/opengl/OpenGLInstance.cpp a/src/qmplay2/opengl/OpenGLInstance.cpp
index e8cef552..b70acfe9 100644
--- b/src/qmplay2/opengl/OpenGLInstance.cpp
+++ a/src/qmplay2/opengl/OpenGLInstance.cpp
@@ -50,9 +50,7 @@ bool OpenGLInstance::init()
         hasPbo = (hasMapBuffer || hasMapBufferRange) && (majorVersion >= (isGLES ? 3 : 2) || extensions.contains("GL_ARB_pixel_buffer_object"));
     }
 
-#ifndef Q_OS_MACOS // On macOS I have always OpenGL 2.1...
     glVer = majorVersion * 10 + context.format().minorVersion();
-#endif
 
     return true;
 }
